src/components/sideNavigation/tests/SideNavigation.spec.tsx

```diff
@@ -0,0 +1,35 @@
+import { shallow, ReactWrapper, mount } from 'enzyme';
+import * as React from 'react';
+import { SideNavigation, ISideNavProps } from '../SideNavigation';
```

Comment:
Logically, your first test should be `should render without errors` and your second one `should render a <SideNavigation /> with classes prop`. In other words, prove your react view renders without errors first, than check what is rendered in the following tests.